### DISCUSSION TITLE
Text Wrap issue

### DISCUSSION THREAD

**Discussion by zuhairmst** [Bodmer/TFT_eSPI#1980] [Q&A - General] [answered]
For strings larger than one line, using text wrap but the remaining text comes at the beginning of next line instead of coming in the centre. What can we do to put it in the centre?

**Comment by Bodmer** [comment, marked as answer]
That is expected behaviour.

To constrain text to a restricted area of the screen use setViewport or split the string in the sketch and draw each part in the desired position.
